wiki/Glossary/Combo_multiplier_effect/id.md

```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - score v1
+  - skor v1
+---
+
+# Efek pengganda kombo
+
+**Efek pengganda kombo** adalah sebuah bug di dalam [scoreV1](/wiki/Score/ScoreV1) yang mana [skor](/wiki/Score) diizinkan untuk dihitung secara mundur. Ini adalah sebuah kecacatan *integer* (bilangan bulat) 32-bit bertanda, di mana (dalam komputasi) *integer* memiliki nilai maksimal `2,147,483,647`. Setelah nilai batasan tersebut tercapai, skor akan mulai dihitung mundur. Di dalam [scoreV2](/wiki/Score#scorev2) masalah ini telah diperbaiki dengan membatasi skor pada 1 juta poin, bukan pengubah akuntansi.
```

Comment:
the word "accounting" in the og article here doesn't literally translate to "akuntansi" orz D:

i'd say if any you can just delete the `,bukan pengubah akuntansi.` part of the sentence here